drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -0,0 +1,873 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ite_it8xxx2_i2c
+
+#include <drivers/i2c.h>
+#include <errno.h>
+#include <logging/log.h>
+		LOG_MODULE_REGISTER(i2c_ite_it8xxx2);
+#include "i2c-priv.h"
+#include <soc.h>
+#include <sys/util.h>
+
+#define DEV_CFG(dev) \
+		((const struct i2c_it8xxx2_config * const)(dev)->config)
+#define DEV_DATA(dev) \
+		((struct i2c_it8xxx2_data * const)(dev)->data)
+
+#define I2C_STANDARD_PORT_COUNT 3
+/* Default PLL frequency. */
+#define PLL_CLOCK 48000000
+
+struct i2c_it8xxx2_config {
+	uint8_t *base;
+	void (*irq_config_func)(void);
+	uint32_t bitrate;
+	uint32_t i2c_irq_base;
+	int port;
```

Comment:
This seems to be an ordinal identifying the port.  `uint8_t` would be more appropriate; it shouldn't be signed.

Reply:
Done